Gestures for BB-8

saraford opened the pull request:
Hi @mintuz! 

Here are a few gestures for BB-8. See gifs below.

I wanted to have the confused sideways glance, but I'm not sure it is possible to move the head to the side, even in the dock. Perhaps someone can figure it out...

I noticed that I have to press enter after the first time I press a key. I copied/pasted the keypress code from drive.js. LMK if you want me to change the instructions to something like "Press any key to begin" to handle this first keystroke.

![bb8-yes2](https://cloud.githubusercontent.com/assets/11529908/17074876/fb6f4698-5037-11e6-8a0a-08b2efe69922.gif)

![bb8-no](https://cloud.githubusercontent.com/assets/11529908/17074879/03145c76-5038-11e6-870f-eed6fcf41e34.gif)

![bb8-likereally](https://cloud.githubusercontent.com/assets/11529908/17074881/12b0fa5e-5038-11e6-9e84-ffbcc670c342.gif)

saraford committed:
added gestures in training.js

saraford committed:
renamed to gestures

saraford committed:
updated for new gesture command

saraford committed:
added comment for y/n gestures source

mintuz commented on commands/gestures.js:
```diff
@@ -0,0 +1,105 @@
+  stdin.on("keypress", function(ch,key) {
+
+    if (key && key.ctrl && key.name === 'c') {
+      process.stdin.pause();
```
before line 19 I think we need to do something like `bb8.disconnect()`

saraford committed:
added disconnect; fixed first keystroke

saraford commented:
@mintuz I've made the following changes:
1. Fixed code so that it works on initial keystroke just like drive.js
2. Added bb8.disconnect() when user quits. 

Thanks!

mintuz commented:
Merging.
